helmfile/charts/gatekeeper/podsecuritypolicies/templates/generic/privilege.yaml

```
@@ -0,0 +1,21 @@
+{{- range $namespace, $services := .Values.constraints }}
+  {{- range $name, $value := $services }}
+    {{- if hasKey $value.allow "privilege" }}
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sPSPPrivilegedContainer
+metadata:
+  name: privilege-{{ $name }}
```

```suggestion
metadata:
  name: privileged-{{ $name }}
```
In order to keep this the same with current PSPs and how SecurityContexts are defined.